CHANGELOG.md

```diff
@@ -24,11 +24,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Admin LTE 2.0 used. Assets files changed.
-- Move ``sonata_wrapper`` block on ```standard_layout.html.twig```
+- Move ``sonata_wrapper`` block on ``standard_layout.html.twig``
```

Comment:
I don't see why we have two backticks too. One is sufficient for the render we need...